chrome/content/zotero/itemTreeColumns.jsx

```diff
@@ -312,29 +315,32 @@ const COLUMNS = [
 		width: "14",
 		minWidth: 14,
 		staticWidth: true,
-		zoteroPersist: new Set(["width", "hidden", "sortDirection"])
+		zoteroPersist: ["width", "hidden", "sortDirection"]
 	},
 	{
 		dataKey: "feed",
 		disabledIn: ["default", "feed"],
 		label: "itemFields.feed",
 		flex: 1,
-		zoteroPersist: new Set(["width", "hidden", "sortDirection"])
+		zoteroPersist: ["width", "hidden", "sortDirection"]
 	}
 ];
 
-function getDefaultColumnByDataKey(dataKey) {
-	return Object.assign({}, COLUMNS.find(col => col.dataKey == dataKey), {hidden: false});
-}
-
-function getDefaultColumnsByDataKeys(dataKeys) {
-	return COLUMNS.filter(column => dataKeys.includes(column.dataKey)).map(column => Object.assign({}, column, {hidden: false}));
+/**
+ * Returns the columns that match the given data keys from the COLUMNS constant.
+ * @param {string | string[]} dataKeys - The data key(s) to match.
+ * @returns {ItemTreeColumnOptions | ItemTreeColumnOptions[]} - The matching columns.
+ */
+function getItemTreeColumnsByDataKeys(dataKeys) {
```

Comment:
@adomasven 
Have it back now. Maybe `getMainItemTreeColumnsByDataKeys`?

Reply:
So once again. These are not main item tree columns. These are column definitions for Zotero Item fields. The function name should be `getColumnDefinitionsByDataKey()`.

Reply:
Done